extensions/wikia/ArticleVideo/ArticleVideoController.class.php

```diff
@@ -5,14 +5,14 @@ public function featured() {
 		$wg = $this->getApp()->wg;
 		$title = $wg->Title->getPrefixedDBkey();
 
-		$enableArticleFeaturedVideo = ArticleVideoContext::isFeaturedVideoEmbedded( $title );
+		$featuredVideoData = ArticleVideoContext::getFeaturedVideoData( $title );
 
-		if ( $enableArticleFeaturedVideo ) {
+		if ( !empty( $featuredVideoData ) ) {
 			$wg->Out->addModules( 'ext.ArticleVideo' );
 
 			// TODO: replace it with DS icon when it's ready (XW-2824)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)